routes/volunteer_hours/volunteer_hours.js

```diff
@@ -0,0 +1,55 @@
+// endpoints related to volunteer hours
+const express = require('express');
+const pool = require('../../db');
+
+const hoursRouter = express();
+
+hoursRouter.use(express.json());
+
+// createSubmittedHours
+hoursRouter.post('/create', async (req, res) => {
+  try {
+    const { user_id, event_id, start_datetime, end_datetime, approved, notes } = req.body;
+    // calculate number of hours
+    const start = new Date(start_datetime);
+    const end = new Date(end_datetime);
+    const diff = end.getTime() - start.getTime();
+    const num_hours = diff / (60000 * 60);
+
+    const createdHours = await pool.query(
+      'INSERT INTO volunteer_hours(user_id, event_id, start_datetime, end_datetime, approved, num_hours, notes, submitted) VALUES($1, $2, $3, $4, $5, $6, $7, $8) RETURNING *;',
+      [user_id, event_id, start_datetime, end_datetime, approved, num_hours, notes, true],
```

Comment:
🚫 **[eslint]** <[camelcase](https://eslint.org/docs/rules/camelcase)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Identifier 'start_datetime' is not in camel case.